back/src/app.ts

```diff
@@ -95,6 +76,18 @@ io.on('connection', function (socket: Socket) {
   });
 });
 
-const server = http.listen(3000, function () {
-  console.log('listening on *:3000');
+const server = http.listen(3000, () => {
```

Comment:
Use envConstants.PORT env variable for this port

Reply:
MMM, wait, what is this port? Why are you connecting database here? We have `app.listen` method to create the Rest API server

Reply:
the Api is listening 8081 and the sockets is working on 3000, it was like that, I already tried with app listen but the sockets is using the http